opentelemetry-api/src/opentelemetry/metrics/_internal/instrument.py

```diff
@@ -361,6 +378,20 @@ def record(
 
 
 class _ProxyHistogram(_ProxyInstrument[Histogram], Histogram):
+    # pylint: disable=super-init-not-called
+    def __init__(
```

Comment:
Is it because of multiple inheritance?

Reply:
I think this is because I haven't updated _ProxyInstrument, fixed.

Reply:
Adding them broke mypy ofc :sweat:

Reply:
Fixed in 0b6fc1f27e2d93b49b9d7fe266f95ba0559cf096